### PR TITLE
[ID1000185] docking

### PR DESCRIPTION
**Pull request by hukunge**
--story=1000185 --user=hukunge
https://www.tapd.cn/60210943/prong/stories/view/1160210943001000185

Signed-off-by: hukunge <hukunge@arxanchain.com>

### CODE REVIEW

**Comment by genggjh** [comment]
为什么在单元测试结果中看不到类似这样的 coverage report？？

2020-03-04 18:14:05.396722 | ub1804go11 | ?   	github.com/arxanchain/asset-mgmt/core/template	[no test files]
2020-03-04 18:14:05.787688 | ub1804go11 | ok  	github.com/arxanchain/asset-mgmt/core/template/impl	0.002s	coverage: 2.1% of statements
2020-03-04 18:14:09.527777 | ub1804go11 | ok  	github.com/arxanchain/asset-mgmt/handler/admin	0.010s	coverage: 6.0% of statements
2020-03-04 18:14:11.554663 | ub1804go11 | ok  	github.com/arxanchain/asset-mgmt/handler/asset	0.014s	coverage: 23.7% of statements

**Review by Nixiangyu — DISMISSED** [review]

File: router/router.go [hunk 1]
```diff
@@ -36,13 +36,13 @@ var (
...
-	urlPubFundsDetail     = "pub/funds/detail"
-	urlPubSuppliesReceive = "pub/supplies/receive"
-	urlPubSuppliesQuery   = "pub/supplies/query"
-	urlPubSuppliesDetail  = "pub/supplies/detail"
-	urlPubList            = "pub/list"
+	urlPubFundsPost      = "pub/funds"
+	urlPubFundsGet       = "pub/funds"
+	urlPubFundsDetail    = "pub/funds/detail"
+	urlPubSuppliesPost   = "pub/supplies"
+	urlPubSuppliesGet    = "pub/supplies"
```
Comment (zhangbing16) [inline router/router.go]: GET和POST可以用一个变量吧？

File: controllers/pub/pub.go [hunk 2]
```diff
@@ -402,7 +417,7 @@ func (h *RestHandler) PubUserList(c *gin.Context) {
 
 	req := &structs.PubUserRequest{}
 	var err error
-	if err = c.BindQuery(req); err != nil {
+	if err = c.Bind(req); err != nil {
```
Comment (zhangbing16) [inline controllers/pub/pub.go:420]: 使用BindQuery接口有问题？

**Comment by zhangbing16** [comment]
git提交和pr的标题太简单了，要描述清楚提交的内容

**Review by zhangbing16 — APPROVED** [review]

**Review by Nixiangyu — APPROVED** [review]

**Pull request merged**